synapse/module_api/__init__.py

```diff
@@ -1386,3 +1396,65 @@ async def remove_room_from_public_room_list(self, room_id: str) -> None:
             room_id: The ID of the room.
         """
         await self._store.set_room_is_public(room_id, False)
+
+
+class AccountDataManager:
```

Comment:
You're making this a separate object just for organization purposes, I assume?

Reply:
Yeah, indeed — a couple of other things do this but otherwise the module API is one large blob and it feels quite disorganised. It may make it hard to split up into multiple files later on.

Reply:
Makes sense, just wanted to make sure there wasn't other magic going on!